tests/unit/test.program.js

```diff
@@ -242,13 +246,31 @@ describe('program.Program', () => {
       });
   });
 
+  it('checks for updates automatically', () => {
+    const root = path.join(__dirname, '..', '..');
+    const handler = spy();
+    const checkForAutomaticUpdates = sinon.stub();
+    const program = new Program(['run'])
+      .command('run', 'some command', handler);
+    return execProgram(program, {
+      checkForUpdates: checkForAutomaticUpdates,
+    })
+      .then(() => {
+        assert.equal(checkForAutomaticUpdates.firstCall.args[0].version,
+                          defaultVersionGetter(root));
```

Comment:
I noticed some test failures locally (surprisingly not on Travis though). I'm going to suggest a few things.

* Stub out `getVersion` for all tests that are not covering any version logic
* Use a stub `getVersion` function in your update checker test. I'm suggesting this because the behavior of `defaultVersionGetter` is already covered in other tests.

Here is a patch to illustrate what I mean:
````diff
diff --git a/tests/unit/test.program.js b/tests/unit/test.program.js
index 58732b8..a4eea98 100644
--- a/tests/unit/test.program.js
+++ b/tests/unit/test.program.js
@@ -21,6 +21,7 @@ describe('program.Program', () => {
     let absolutePackageDir = path.join(__dirname, '..', '..');
     return program.execute(
       absolutePackageDir, {
+        getVersion: () => 'not-a-real-version',
         checkForUpdates: spy(),
         systemProcess: fakeProcess,
         shouldExitProgram: false,
@@ -251,12 +252,14 @@ describe('program.Program', () => {
     const checkForAutomaticUpdates = sinon.stub();
     const program = new Program(['run'])
       .command('run', 'some command', handler);
+    const getVersion = () => 'some-package-version';
     return execProgram(program, {
+      getVersion,
       checkForUpdates: checkForAutomaticUpdates,
     })
       .then(() => {
         assert.equal(checkForAutomaticUpdates.firstCall.args[0].version,
-                          defaultVersionGetter(root));
+                     'some-package-version');
       });
   });
 });
@@ -266,6 +269,7 @@ describe('program.main', () => {
 
   function execProgram(argv, {projectRoot = '', ...mainOptions}: Object = {}) {
     const runOptions = {
+      getVersion: () => 'not-a-real-version',
       checkForUpdates: spy(),
       shouldExitProgram: false,
       systemProcess: fake(process),
````